ui/src/app/components/runs/runs.component.ts

```diff
@@ -96,7 +98,7 @@ export class RunsComponent implements OnDestroy, AfterViewInit {
   }
 
   clearSort() {
-    !!this.sort ? this.columns.find(_ => _.field == this.sort.by).sortOrder = 0 : undefined;
+    !!this.sort ? this.columns.find(_ => _.field === this.sort.by).sortOrder = 0 : undefined;
```

Comment:
You are right about this one, but in the future please don't mix in code which is unrelated to the issue. It makes the review very time-consuming and one PR should deal with only one issue. Changing code unrelated to your issue also pose the risk of causing merge-conflicts with other open PRs.
It's better to open an issue for that which will be solved another time.

Reply:
noted with thanks :)